envoy/api/v2/route/route.proto

```diff
@@ -208,6 +208,26 @@ message WeightedCluster {
     // cluster with metadata matching that set in metadata_match will be
     // considered. The filter name should be specified as *envoy.lb*.
     Metadata metadata_match = 3;
+
+    // Specifies a list of headers to be added to requests when this cluster is selected
```

Comment:
Please add the not implemented tags unless the associated Envoy PR is going to get posted at the same time.